src/network/networkd-link.c

```diff
@@ -1369,6 +1366,11 @@ static int link_set_flags(Link *link) {
                 ifi_flags |= link->network->arp ? 0 : IFF_NOARP;
```

Comment:
event though not strictly necessary I figure this should use the SET_FLAG macro really